practices/java-web/HelloWorldServletMvcWebApplication/src/main/java/com/dowa/java/web/HelloWorldMvcServlet.java

```diff
@@ -21,10 +24,27 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws Se
     protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
 
         RequestDispatcher requestDispatcher = req.getRequestDispatcher("/Hello.jsp");
+        RequestDispatcher requestDispatcher1 = req.getRequestDispatcher("/Error.jsp");
```

Comment:
why different RequestDispatcher objects? the only thing that changes is the url so that what's should be on a variable